internal/controllers/machinehealthcheck/machinehealthcheck_targets.go

```diff
@@ -97,6 +98,13 @@ func (t *healthCheckTarget) needsRemediation(logger logr.Logger, timeoutForMachi
 	if annotations.HasRemediateMachine(t.Machine) {
 		conditions.MarkFalse(t.Machine, clusterv1.MachineHealthCheckSucceededCondition, clusterv1.HasRemediateMachineAnnotationReason, clusterv1.ConditionSeverityWarning, "Marked for remediation via remediate-machine annotation")
 		logger.V(3).Info("Target is marked for remediation via remediate-machine annotation")
+
+		v1beta2conditions.Set(t.Machine, metav1.Condition{
+			Type:    clusterv1.MachineHealthCheckSucceededV1Beta2Condition,
+			Status:  metav1.ConditionFalse,
+			Reason:  clusterv1.MachineHealthCheckManuallyRemediatedV1Beta2Reason,
+			Message: "Marked for remediation via remediate-machine annotation",
+		})
 		return true, time.Duration(0)
 	}
 
```

Comment:
Just to double check. I assume we don't handle the next two cases because at the point where we actually act on the OwnerRemediated conditions failureReason & failureMessage will be gone?

Reply:
That's correct